### ISSUE TITLE
Base model training issue

### ISSUE THREAD

**Issue by ParinazRZ** [wang-qiuchen/PseDet#6]
Hi,

I read your paper and I'm very interested in using it for my research. Great work! 

I used the exact code in ./coco_cfg/train_stage_init/gfl_r50_1x_s0_40cls.py as configuration and tried to reproduce your results; however, after 12 epochs, the model is not training. I am using one A100 GPU for training but other than that, the batch size, learning rate, and all other details in the gfl_r50_1x_s0_40cls.py are being used without any changes. I am using the pre-trained torchvision ResNet as suggested (https://download.pytorch.org/models/resnet50-0676ba61.pth) along with the COCO 2017 database. 

Should the batch size be 2 per GPU? So, to reproduce your results with 1 GPU, I need to adjust it to 16? Is the learning rate set to align with batch size = 2?

**Comment by wang-qiuchen** [comment]
sorry，i have no idea about the parameters used in single GPU training，maybe you can use single gpu to reproduce the official gfl code and confirm the effectiveness，then implement the setting to PseDet
